content/md/assets/family-by-family-pam-migration.md

```diff
@@ -0,0 +1,104 @@
+---
+id: family-by-family-pam-migration
+title: The **family-by-family** migration guide for PAM assets
+popular: false
+ee-only: true
+related: from-PAM-to-new-AM, how-to-migrate-from-pam
+---
+
+# What's the family-by-family migration?
+In this migration, you can have **several asset families** for your existing PAM assets.
+
+You will need to define manually 2 things:
+- For each asset, its asset family.
+- For each asset collection product attribute, the asset family linked.
+
+Other steps are automated, you just need to launch few commands and your PAM assets will be migrated in the 4.0 new Asset Manager:
+- The asset families will be created automatically in the PIM.
+- All your assets will be migrated and linked to the asset family you defined.
+- All you existing PAM asset collection product attributes will be linked to the asset family.
+- And all your links between assets and products / product models will be kept.
+
+# What will my asset families gonna look like?
+The asset family created will contain the following attributes:
+- `reference` (*media_file attribute*): containing the reference file of the PAM asset and used as attribute as main media for this family
+- `reference_localizable` (*media_file attribute*): an attribute with a value per locale containing the reference file of the asset for each locale, if the asset was localizable in the PAM
+- `variation_scopable` (*media_file attribute*): an attribute with a value per channel containing the asset variation file for each channel
+- `variation_scopable_localizable` (*media_file attribute*): an attribute with a value per channel containing the asset variation file for each channel and locale, if the asset was localizable in the PAM
+- `description` (*text attribute*):  with the PAM asset description
+- `categories` (*text attribute*): with the PAM asset categories separated by a comma
+- `tags` (*text attribute*): with the PAM asset tags (tags are separated by a comma)
+- `end_of_used`: text attribute with the asset end of use date
+
+![PAM assets family](pam-assets-family.png)
+
+# What will my assets gonna look like?
+
+![PAM asset](pam-asset.png)
+
+# What are the steps to migrate?
+Don't be afraid, this part is a bit technical. If you are a Julia, you can ask your IT guy or your integrator to do it.
+
+<!-- TOREVIEW
+
+## _Step 1_ | Export the PAM assets
+First, we need to export your PAM assets with all their data (properties, reference and variation files) in CSV files.
+To do so, in the PIM:
+1. Run `export-pam-assets`
+2. Put the CSV files in `tmp` folder.
+
+## _Step 2_ | Split the PAM assets family by family
+Then, as you want to create several asset families for your assets, you need to split the asset files family by family.
```

Comment:
We will improve this part, this needs to be more detailed

Reply:
Oui, j'avais commenté cette partie-là car on ne sait pas encore jusqu'où on va aller